rclip/model.py

```diff
@@ -183,11 +184,14 @@ def compute_features_for_queries(self, queries: List[str]) -> FeatureVector:
       url_multipliers, url_paths = cast(Tuple[Tuple[float], Tuple[str]], zip(*(urls))) if urls else ((), ())
       try:
         images = [helpers.download_image(q) for q in url_paths] + [helpers.read_image(q) for q in file_paths]
+      except DecompressionBombError as e:
+        print(e, file=sys.stderr)
```

Comment:
Maybe we should add a comment that `e` is a custom one – with an already formatted message, prepared in `.read_image` and `.download_image`.